PowerValidatedSolutions.psm1

```diff
@@ -21799,6 +21799,12 @@ Function Test-InvPrerequisite {
                         Test-PrereqBinaryNetworks -searchCriteria "VMware-Aria-Operations-for-Networks-$ariaOperationsForNetworksVersion" # Verify that the required binaries are available
                         Test-PrereqLicenseKey -licenseKey $jsonInput.licenseKey -productName "VMware Aria Suite or VMware Aria Operations for Networks" # Verify a license key is present
                         Test-PrereqDomainController -server ($jsonInput.domainControllerMachineName + "." + $jsonInput.domainFqdn) # Verify that Active Directory Domain Controllers are available in the environment
+                        # Verify that DNS Entries are resolvable in the environment
+                        $dnsEntries = '["'+ $jsonInput.ariaNetworksPlatformNodeaHostname + '.' + $jsonInput.domain + '", "'+ $jsonInput.ariaNetworksCollectorNodeaHostname + '.' + $jsonInput.domainProxies + '"]' | ConvertFrom-Json
```

Comment:
Why not just use $jsonInput.ariaNetworksPlatformNodeaFqdn and $jsonInput.ariaNetworksCollectorNodeaFqdn?

Reply:
Udpated to use those variables.

Reply:
Updated In https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/839b36abdacf09ae8380c818b8ebaab40f61f34a